badge-maker/lib/badge-renderers.js

```diff
@@ -82,22 +77,19 @@ function renderLogo({
   logoWidth = 14,
   logoPadding = 0,
 }) {
-  if (!logo) {
+  if (logo) {
+    const logoHeight = 14
+    const y = (badgeHeight - logoHeight) / 2
+    const x = horizPadding
     return {
-      hasLogo: false,
-      totalLogoWidth: 0,
-      renderedLogo: '',
+      hasLogo: true,
+      totalLogoWidth: logoWidth + logoPadding,
+      renderedLogo: `<image x="${x}" y="${y}" width="${logoWidth}" height="14" xlink:href="${escapeXml(
```

Comment:
Not strictly related to the changes here, but is the height attribute intended to be aligned with `logoHeight` or do they both just coincidentally have a value of `14`? If the former, is there any reason why the attribute has an inline value instead of using the variable?

Reply:
Yes, I suspect it probably should be using `${logoHeight}`. It seems okay to make that change here.